DEVELOPMENT_FLOW.md

```diff
@@ -0,0 +1,32 @@
+# OPI Project Development Flow
+
+This document attempts to document the development flow as the project moves
+towards code generation.
+
+## Sub-Projects and GitHub Repositories
+
+Each sub-project has their own GitHub Repository where they are currently
+iterating on documents and some early code. We advocate that model should
+continue going forward. Further, if a sub-project wants to create additional
+GitHub repositories, that should be fine as well.
```

Comment:
any naming conventions if we want to create more repos for the same sub-group ?

Reply:
I don't think we need to necessarily tie the name of a repo to the subgroup that initially created it.  Each repo should be accurately named for its purpose.

Reply:
I mean, not that I advocate for it, like mandatory `opi-` prefix on each repo name